### PR TITLE
Introduce keytableprinter.FormatKeyWarningLines

### PR DESCRIPTION
**Pull request by idrysdale**
FormatKeyWarningLines takes a status.KeyWarning and returns an array of human friendly messages coloured appropriately for printing to the terminal.

### CODE REVIEW

**Comment by paulfurley** [comment]
erg finally reviewing now!

File: keytableprinter/formatwarnings_test.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		},
+		{
+			status.OverdueForRotation{DaysUntilExpiry: 5},
+			[]string{
+				colour.Red("Overdue for rotation ⏰"),
+				colour.Red("Expires in 5 days!"),
+			},
+		},
+		{
+			status.OverdueForRotation{DaysUntilExpiry: 1},
```
Comment (paulfurley) [inline keytableprinter/formatwarnings_test.go]: beautiful

File: keytableprinter/formatwarnings_test.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+			[]string{
+				colour.Red("Overdue for rotation ⏰"),
+				colour.Red("Expires tomorrow!"),
+			},
+		},
+		{
+			status.OverdueForRotation{DaysUntilExpiry: 0},
```
Comment (paulfurley) [inline keytableprinter/formatwarnings_test.go]: i wish expirybot had this much love and attention

**Comment by paulfurley** [comment]
I don't understand the mixing of `Warn`, `Grey` and `Red`, ... `Warn` is different type of thing than `Grey` and `Red`, so I'm not too sure how they're going to look

**Comment by paulfurley** [comment]
Apart from the colours thing, this looks great!

**Comment by idrysdale** [comment]
Cool, good shout. I'll resolve the colouring. Nice one!

**Comment by paulfurley** [comment]
Boom!

**Pull request merged**